src/cli/tc.rs

```diff
@@ -104,11 +108,37 @@ pub async fn init(file_path: &str) -> Result<()> {
 pub fn display_schema(blueprint: &Blueprint) {
   Fmt::display(Fmt::heading(&"GraphQL Schema:\n".to_string()));
   let sdl = blueprint.to_schema();
-  Fmt::display(print_schema::print_schema(sdl));
+  Fmt::display(format!("{}\n", print_schema::print_schema(sdl)));
 }
 
 fn display_config(config: &Config, n_plus_one_queries: bool) {
-  Fmt::display(Fmt::success(&"No errors found".to_string()));
```

Comment:
We need to restore this; the message should be changed but not deleted

```suggestion
  Fmt::display(Fmt::success(&"No errors found on config".to_string()));
```

Reply:
this review is still valid @tobihans

Reply:
this review is still valid @tobihans. IMO, we shouldn't delete this line.

Reply:
I think we can convert this to a log message instead. So that people have an option to enable/disable it via env variables.